alfasim_sdk/hook_specs_gui.py

```diff
@@ -1,10 +1,17 @@
 from pluggy import HookspecMarker
 
-hookspec = HookspecMarker("alfasim")
+hookspec = HookspecMarker("ALFAsim")
 
 
 @hookspec
 def alfasim_get_data_model_type():
     """
-    Entry point for the creation of models in Alfasim
+    Entry point for the creation of models in ALFAsim
+    """
+
+
+@hookspec
+def alfasim_get_additional_variables():
+    """
+    Add additional variables in ALFAsim
```

Comment:
Improve this documentation a little bit, as it is it is very vague

Reply:
What do you think of this docstring:
````python
@hookspec
def alfasim_get_additional_variables():
    """
    Allows plugins to register new additional variables on ALFAsim.
    This variable can be used to store internal data from the plugin,
    or it can be used to expose data to the user in the plot window.
    """
```